app/controllers/integrations/buildkite_controller.rb

```diff
@@ -28,4 +28,9 @@ def build_param
   def message
     build_param[:message]
   end
+
+  def release_params
+    extra_params = Samson::Hooks.fire(:release_params, project, build_param)
```

Comment:
if it's only sued here it cannot be `release_params` but must be `buildkite_release_params` or so ... but might be easier to add a buildkite_controller_decorator ... not sure if that is supported ... but adding support for controller hacks this way could be work ...